From a8ef68d4fbd678c3f9ad8506bae053777dee22f7 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Wed, 21 Mar 2018 13:09:59 +0000
Subject: [PATCH] OS-6800 bhyve-opts should be bhyve-extra-opts Reviewed by:
 Mike Gerdts <mike.gerdts@joyent.com> Approved by: Mike Gerdts
 <mike.gerdts@joyent.com>

---
 src/vm/node_modules/VM.js        | 3 +--
 src/vm/node_modules/proptable.js | 4 ++--
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/src/vm/node_modules/VM.js b/src/vm/node_modules/VM.js
index 09a53ef3..3c65bb78 100644
--- a/src/vm/node_modules/VM.js
+++ b/src/vm/node_modules/VM.js
@@ -6086,7 +6086,7 @@ function buildZonecfgUpdate(vmobj, payload, log)
         setAttr('com1', 'com1');
         setAttr('com2', 'com2');
 
-        setAttr('bhyve-opts', 'bhyve_opts');
+        setAttr('bhyve-extra-opts', 'bhyve_extra_opts');
     }
 
     // NOTE: Thanks to normalizePayload() we'll only have these when relevant
@@ -12002,7 +12002,6 @@ exports.start = function (uuid, extra, options, callback)
             'nics',
             'platform_buildstamp',
             'bhyve_extra_opts',
-            'bhyve_opts',
             'ram',
             'resolvers',
             'state',
diff --git a/src/vm/node_modules/proptable.js b/src/vm/node_modules/proptable.js
index 7bbb6c5e..fd698311 100644
--- a/src/vm/node_modules/proptable.js
+++ b/src/vm/node_modules/proptable.js
@@ -325,14 +325,14 @@ exports.properties = {
             type: 'boolean'
         },
         zonexml: 'zone.autoboot'
-    }, bhyve_opts: {
+    }, bhyve_extra_opts: {
         payload: {
             allowed: {
                 'bhyve': ['create', 'receive', 'update']
             },
             type: 'string'
         },
-        zonexml: 'zone.attr.bhyve-opts'
+        zonexml: 'zone.attr.bhyve-extra-opts'
     }, billing_id: {
         payload: {
             allowed: {
-- 
2.21.0

